atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/StreamStoreRenderer.java

```diff
@@ -688,8 +689,11 @@ private void touchMetadataWhileMarkingUsedForConflicts() {
                     line("for (Map.Entry<", StreamMetadataRow, ", StreamMetadata> e : metadatas.entrySet()) {");
                     {
                         line("StreamMetadata metadata = e.getValue();");
-                        line("Preconditions.checkState(metadata.getStatus() == Status.STORED,");
-                        line("\"Stream: %s has status: %s\", e.getKey().getId(), metadata.getStatus());");
+                        line("Preconditions.checkState(");
+                        line("        metadata.getStatus() == Status.STORED,");
+                        line("        \"Stream has stored status\",");
+                        line("        SafeArg.of(\"streamId\", e.getKey().getId()),");
+                        line("        SafeArg.of(\"status\", metadata.getStatus()));");
```

Comment:
similar question as above around safe/unsafe for `streamId` which should be an opaque identifier, and `status` is enum so should be safe

Reply:
`streamId` is a timestamp, so safe.